docs/integrations/saas-cloud/docusign.md

```diff
@@ -0,0 +1,210 @@
+     "9cc9a7f1-a853-45ab-95c1-22c7e7834e1d",
+     "aaf22d31-9afa-48f3-880e-6287e4179daf",
+     "b4cc9614-6606-4fd9-86df-fad67b2f6181",
+     "de8ebe47-9e94-42e4-86db-4d56f882ab0a",
+     "e9e27037-213f-46db-9976-2b22c1b36bc9",
+     "f767e823-9722-49ef-8da5-b0d9f8dd13b6"
+   ],
+   "Count": "18"
+ }
+}
+```
+
+### Sample Queries
+
+This section contains the sample queries of both the `Events` and `Alerts`.
+
+```sql title="User Access Controls"
+_sourceCategory="docusign_src"
+| json "object","userId","eventId","action","property","source","ipAddressLocation.latitude","ipAddressLocation.longitude","result","ipAddressLocation.city","ipAddressLocation.state","ipAddressLocation.country","data" as object,user_id,event_id,action,property,source,latitude,longitude,result,city,state,country,data nodrop
+| where object matches "{{object}}" and action matches"{{action}}" and source matches"{{source}}"
+| where country matches "{{country}}" or isNull(country)
+| where object matches("*Account*") or object matches("*Connect*") or object matches("*User*") or object matches("*PermissionSet*")
+| where action matches("*Updated*")
+| count_distinct(event_id) by _messageTime,object,action,property,data,source,result,city,state,country
+| formatDate(toLong(_messageTime), "dd-MM-yyyy HH:mm:ss") as time
+| if(isBlank(property),"nil",property) as property
+| top 100 time,object,action,property,data,source,result,city,state,country by time
+```
+
+```sql title="Alerts by Severity Over Time"
+_sourceCategory="docusign_src"
+| json "object","userId","eventId","action","property","source","ipAddressLocation.latitude","ipAddressLocation.longitude","result","ipAddressLocation.city","ipAddressLocation.state","ipAddressLocation.country" as object,user_id,event_id,action,property,source,latitude,longitude,result,city,state,country nodrop
+| where object matches "{{object}}" and action matches"{{action}}" and source matches"{{source}}"
+| where country matches "{{country}}" or isNull(country)
+| where object matches("*Alert*")
+| lookup alert,object,ds_action,description,severity,severity_id from https://sumologic-app-data.s3.amazonaws.com/docusign_alert_lookup.csv on action = ds_action
+| if(isNull(severity),"low",severity) as severity
+| where severity matches "{{severity}}"
+| timeslice 1d
+| count_distinct(event_id) as frequency by _timeslice,severity
+| fillmissing timeslice,values all in severity
+| transpose row _timeslice column severity as low, medium, high
+```
+
+## Installing the DocuSign App
+
+Locate and install the app from the **App Catalog**. To preview the dashboards included with the app before installing, click **Preview Dashboards**.
+
+Before you begin, collect logs from DocuSign API and ingest them into Sumo Logic. Refer to the [DocuSign API Cloud-to-Cloud Integration](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/) to create the source and use the same source category while installing the app.
```

Comment:
This path doesn't exist. /docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source/)